datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -75,7 +75,7 @@ pysnmp==4.4.9
 pysocks==1.7.1
 python-binary-memcached==0.26.1; sys_platform != 'win32' and python_version < '3.0'
 python-binary-memcached==0.31.2; sys_platform != 'win32' and python_version > '3.0'
-python-dateutil==2.8.2
+python-dateutil==2.9.0.post0
```

Comment:
Interesting versioning. I gave this one a double take 😆 but looks valid- https://pypi.org/project/python-dateutil/.